app.js

```diff
@@ -3,14 +3,31 @@ const app = express();
 
 const { getTopics } = require("./controllers/topics-controllers");
 const { getDocs } = require("./controllers/api-constrollers");
+const { getArticleById } = require("./controllers/articles-controllers");
 
 app.get("/api", getDocs);
 
 app.get("/api/topics", getTopics);
 
+app.get("/api/articles/:article_id", getArticleById);
+
 app.use((req, res, next) => {
   res.status(404).send({ msg: "Route not found" });
   next(err);
 });
 
+app.use((err, req, res, next) => {
```

Comment:
You only need 1 error handler for custom errors. Just check for a status and message and then pass those values through. res.status(err.status).send({msg: err.msg})